components/brave_ads/browser/features.cc

```diff
@@ -0,0 +1,149 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/brave_ads/browser/features.h"
+
+#include "base/feature_list.h"
+#include "build/build_config.h"
+
+namespace brave_ads {
+namespace features {
+
+const base::Feature kAdNotifications{"AdNotifications",
+                                     base::FEATURE_ENABLED_BY_DEFAULT};
+
+namespace {
+
+// Set to true to show custom ad notifications or false to show system
+// notifications
+const char kFieldTrialParameterShouldShowCustomAdNotifications[] =
+    "should_show_custom_notifications";
+const int kDefaultShouldShowCustomAdNotifications = true;
```

Comment:
Should this be boolean?
```suggestion
const bool kDefaultShouldShowCustomAdNotifications = true;
```

Reply:
Great spot :-)

Reply:
Fixed